src/workerd/api/container.c++

```diff
@@ -75,7 +94,7 @@ jsg::Promise<void> Container::monitor(jsg::Lock& js) {
     running = false;
```

Comment:
You can use this block which is already the error handler 😄 

Would really help deduplicate some code.